src/classes/STG_PanelHealthCheck_CTRL.cls

```diff
@@ -62,6 +62,44 @@ public without sharing class STG_PanelHealthCheck_CTRL extends STG_Panel {
         } private set;
     }
 
+    /*********************************************************************************************************
+    * @description Override saveSettings from STG_Panel to update Custom Setting Error Setting
```

Comment:
Let's provide more details what the Error Settings update impacts. It is not clear from reading this method how the Error settings is related to the Health Check configuration.